### DISCUSSION TITLE
How to remove upgrade icon?

### DISCUSSION THREAD

**Discussion by kingofdice66** [tinymce/tinymce#8152] [General]
How to remove the upgrade icon marked with a red arrow on top right corner in the picture?

![github](https://user-images.githubusercontent.com/50295155/192414867-2c69f9ca-0ad6-4f29-9b6c-6602514b9086.PNG)

**Comment by brian-forte** [comment]
From the TinyMCE documentation: [Premium upgrade promotion](https://tiny.cloud/docs/tinymce/6/editor-premium-upgrade-promotion/)

**Reply by kingofdice66** [reply]
Thank you!
